exporter/src/pool/collector/pools_collector.rs

```diff
@@ -112,6 +123,20 @@ impl Collector for PoolsCollector {
             let mut x = pool_used_size.collect();
             metric_family.extend(x.pop());
 
+            let pool_committed_size = match self
+                .pool_committed_size
+                .get_metric_with_label_values(&[node_name.clone().as_str(), p.name().as_str()])
+            {
+                Ok(pool_committed_size) => pool_committed_size,
+                Err(error) => {
+                    error!(%error,"Error while creating metrics(pool_used_size) with label values:");
```

Comment:
There's a colon here, missing value?

Reply:
Yes copy&paste from existing code, I fixed that the existing ones.